UI/src/com/infiniteautomation/mango/rest/latest/BootstrapController.java

```diff
@@ -215,6 +225,16 @@ public PostLoginData postLogin(@AuthenticationPrincipal PermissionHolder user) {
 
         data.setTranslations(TranslationsController.getTranslations(PRIVATE_TRANSLATIONS, user.getLocaleObject()));
 
+        var aggregateDao = pointValueDao.getAggregateDao();
+        var aggregationEnabled = aggregateDao.supportsPreAggregation() && aggregateDao.isPreAggregationEnabled();
+        data.setAggregationEnabled(aggregationEnabled);
+
+        if (aggregateDao instanceof BoundaryAggregateDao) {
+            var boundaryDate = ((BoundaryAggregateDao) aggregateDao).boundary();
+            var queryBoundary = ChronoUnit.MILLIS.between(boundaryDate, ZonedDateTime.now(clock));
+            data.setQueryBoundary(queryBoundary);
```

Comment:
It would be better to pass the boundary directly to UI in the model, the bootstrap data is only fetched once on first load and is cached.